### ISSUE TITLE
Cannot Disable Location Polling

### ISSUE THREAD

**Issue by Jericon** [dgreif/ring#561]
# Bug Report

### Describe the Bug
 Location Mode Polling can be disabled by setting the interval to 0, however, when setting 0, the UI says it is an invalid argument.

### To Reproduce
 - Go to Homebridge UI
 - Go to homebridge-ring settings
 - Scroll to "Location Mode Polling"
 - See text that says "The Location Mode switch can be hidden by setting this option to 0"
 - Set value to 0
 - Error occurs

### Expected behavior
 0 Should be allowed to be entered.

### Screenshots/Logs
 If applicable, add screenshots and log output to help explain your problem.

### Additional context
 Add any other context about the problem here.

### Homebridge Ring Config
Post homebridge-ring platform config  _without sensitive information_
```
        {
            "refreshToken": "<REDACTED>",
            "hideDoorbellSwitch": true,
            "hideUnsupportedServices": true,
            "sendDoorbellMotionNotificationsToTv": true,
            "cameraDingsPollingSeconds": 5,
            "platform": "Ring"
        },
```

### Environment

 - OS: [e.g. Raspbian] Raspbian
 - Node.js: [e.g. 12.14.1] 12.18.3
 - NPM: [e.g 6.13.4] 6.14.8
 - homebridge-ring: [e.g 6.0.3] 9.14.1
 - homebridge: [e.g. 0.4.50 or N/A] 1.1.7
 - hoobs: [e.g. 0.4.50 or N/A] NA
 - Homebridge-UI Version: 4.36.0

**Comment by Jericon** [comment]
Setting the value to a "valid" number, then changing the JSON config to 0 does work.

**Comment by dgreif** [comment]
Fixed in `9.14.2`.  Thanks for reporting!
